setup.cfg

```diff
@@ -87,7 +87,7 @@ all =
     pytz
     jplephem
     mpmath
-    asdf>=2.10.0
+    asdf-astropy @ git+https://github.com/eslavich/asdf-astropy.git@eslavich-add-astropy-table-plugin
```

Comment:
Note to other reviewers: This pin needs to be updated to a stable minversion pin before this PR can be merged.